tests/scalers/pause-at-n.test.ts

```diff
@@ -0,0 +1,199 @@
+import * as async from 'async'
+import * as fs from 'fs'
+import * as azure from 'azure-storage'
+import * as sh from 'shelljs'
+import * as tmp from 'tmp'
+import test from 'ava'
+import {createNamespace, waitForDeploymentReplicaCount} from "./helpers";
+
+const deploymentFile = tmp.fileSync()
+const scaledObjectFile = tmp.fileSync()
+const scaledObjectFileWithAnnotation = tmp.fileSync()
+const pauseCount = 5
+
+const testNamespace = 'pause-at-n-test'
+const queueName = 'paused-replicas-queue-name'
```

Comment:
We shouldn't use the same queue name as in the other test. Tests run in parallel so shouldn't share resources.